csm/cli/cortxcli.py

```diff
@@ -100,6 +102,25 @@ def get_password(value, confirm_pass_flag=True):
                 raise ArgumentError(errno.EINVAL, "Password do not match.")
         return value
 
+    @staticmethod
+    def get_username(value, confirm_username_flag=True):
```

Comment:
get_username does not clearly states which username. Rename the method to be more specific

Reply:
Changes done.